### ISSUE TITLE
Create Simple Example Application

### ISSUE THREAD

**Issue by payton** [payton/django-siwe-auth#12]
Create a second example application with much simpler implementation.

This should just be a login screen and success screen.

**Comment by pconerly** [comment]
This task was completed from `examples/notepad`, right?

**Comment by payton** [comment]
@pconerly no, I wanted to also add a bare bones example. No styling, two buttons, very basic.
